### PR TITLE
Closes BG-1059, BG-1060: Handle new unclaimed endowments

### PR DESCRIPTION
**Pull request by ap-justin**
## Explanation of the solution
* reflect unclaimed endow type
* update UI to handle type changes ( see dummy unclaimed endow:88) 
* add verification status filters
image

* claimed endowment's card has verified badge
image

* claimed endowment's profile has verified badge
image

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/types/aws/ap/index.ts [hunk 1]
```diff
@@ -49,12 +49,15 @@ export type Endowment = {
   endow_designation: EndowDesignation;
   fiscal_sponsored: boolean;
   hq_country: string;
+  /** empty string by default */
```
Comment (ap-justin) [inline src/types/aws/ap/index.ts:52]: assets are set to `""` by default after admin approvals

File: src/pages/Marketplace/ActiveFilters.tsx [hunk 2]
```diff
@@ -89,7 +101,7 @@ function Item({ children, onRemove }: PropsWithChildren<{ onRemove(): void }>) {
     <button
       type="button"
       onClick={onRemove}
-      className="flex items-center gap-2 border select-none rounded-full cursor-pointer capitalize text-xs pt-1 pb-[.3rem] pl-3 pr-1.5 texg-gray-d1 dark:text-navy-l2 bg-blue-l5 hover:bg-blue-l4"
+      className="flex items-center gap-2 border select-none rounded-full cursor-pointer capitalize text-xs py-1 pl-3 pr-1.5 texg-gray-d1 dark:text-navy-l2 bg-blue-l5 hover:bg-blue-l4"
```
Comment (ap-justin) [inline src/pages/Marketplace/ActiveFilters.tsx:104]:
```suggestion
      className="flex items-center gap-2 border select-none rounded-full cursor-pointer capitalize text-xs pt-1 pb-[.3rem] pl-3 pr-1.5 texg-gray-d1 dark:text-navy-l2 bg-blue-l5 hover:bg-blue-l4"
```
different top bottom padding was used before to compensate differing line height of font `Open Sans` to vertically center text.

File: src/components/Tooltip.tsx [hunk 3]
```diff
@@ -17,17 +17,20 @@ export default function Tooltip<T extends HTMLElement>({
   const [isHovered, setHovered] = useState(false);
 
   useEffect(() => {
-    if (!anchorRef?.current) {
-      return;
+    const element = anchorRef?.current;
+    if (!element) return;
+
+    function handleMouseEnter(this: HTMLElement) {
+      const rect = this.getBoundingClientRect();
```
Comment (ap-justin) [inline src/components/Tooltip.tsx:24]: now compute rec client when user hovers

**Review by SovereignAndrey — APPROVED** [review]

File: src/components/Tooltip.tsx (hunk 3 above)
Comment (misicnenad) [reply src/components/Tooltip.tsx:24]: @ap-justin @SovereignAndrey there's this weird edge case behavior that remained - if a user hovers the icon and immediately scrolls down/up, the tooltip follows the page scroll (see video below).

Not sure if this is a big deal, but it's there.

https://www.loom.com/share/3b4526712e064f09aab2d7cb715fe5cd?sid=d9eca95c-c267-4e61-8e9c-ccdb7c6a32e5

**Pull request merged**